admin-app/src/main/kotlin/com/epam/drill/admin/config/DatabaseConfig.kt

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.admin.config
+
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import io.ktor.application.*
+import io.ktor.config.*
+import org.kodein.di.DI
+import org.kodein.di.bind
+import org.kodein.di.instance
+import org.kodein.di.singleton
+import javax.sql.DataSource
+
+class DatabaseConfig(private val config: ApplicationConfig) {
+    val host: String
+        get() = config.propertyOrNull("host")?.getString() ?: "localhost"
```

Comment:
As far as I remember, we intentionally decided not to create defaults "everywhere", and instead keep default values to a single configuration file - `application.conf`

Reply:
We had overcomplications with `application.conf` due to Auth impl - it is no longer a single source of truth